### ISSUE TITLE
No way to edit a login

### ISSUE THREAD

**Issue by Cybrosys** [microsoft/azuredatastudio#10080]
Issue Type: <b>Bug</b>

Hello,

There does not seem to by a way of editing a login. When I right-click a login I only have the option to "Refresh".

Azure Data Studio version: azuredatastudio 1.16.1 (16801c522df68e63218b39ccb5b4a7431f9bf667, 2020-03-23T22:06:16.901Z)
OS version: Darwin x64 19.4.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|32.00GB (0.60GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by aaomidi** [comment]
Do you mean you want to login with a different account?

You can press the + sign on the Azure pane:

![image](https://user-images.githubusercontent.com/1465785/79790140-260f5880-8300-11ea-9ed7-3b34a8086e96.png)

and then you should be able to delete the account here:

![image](https://user-images.githubusercontent.com/1465785/79790164-2f98c080-8300-11ea-93d6-13077cef808f.png)

**Comment by Cybrosys** [comment]
No, I mean I want to edit the Security Logins configured for the SQL Server that I am connected to.
